lib/tasks/build.js

```diff
@@ -33,19 +37,11 @@ module.exports.js = function(gulp, config) {
 	config = config || {};
 	var src = config.js || files.getMainJsPath() || null;
 	if (src) {
-		var transforms = ['babelify', 'debowerify', 'textrequireify'].concat(config.transforms || []);
-		delete config.transforms;
-
-		// Hackily shallow clone the config object
```

Comment:
@matthew-andrews  It seems you added this, I can't see an obvious reason to keep it.  Can you remember why it was needed exactly?